news/docs-elaborate-parse-phases.rst

```diff
@@ -0,0 +1,14 @@
+**Added:**
+
+* Docs to the execer parse phase.
+
```

Comment:
Does this need a news item? Also this wont get pushed in the html docs that the user normally sees.

Reply:
I'm not sure - I just added it because it was required for the CI?

Reply:
You can remove remove it in this case.